docs/get-started.md

```diff
@@ -4,31 +4,29 @@ title: Temporal documentation
 sidebar_label: Get started
 ---
 
-Welcome to Temporal's user documentation!
-
 Temporal reimagines state-dependent service-orchestrated application development, and so much more.
 
 ## Get started
 
-Ready to develop? [Install the Temporal server](install-temporal-server) and choose your SDK.
-
-→ [Go SDK](go-quick-start)
+Ready to build? [Install the Temporal server](install-temporal-server) and choose your SDK.
 
-→ [Java SDK](java-quick-start)
+- [Go SDK](go-quick-start)
+- [Java SDK](java-quick-start)
 
 ## Learn about Temporal
 
-Building applications with Temporal is a paradigm shift. If you are new to Temporal, or workflow applications in general, get caught up on the [core concepts](concepts-overview) and common [use cases](use-cases-overview).
+Building applications with Temporal is a paradigm shift. If you are new to Temporal, or workflow applications in general, get caught up on the [core concepts](concepts-overview).
```

Comment:
Still not a fan of starting out with "paradigm shift". Sound daunting. Start with the positive about how we will make it so much easier etc.